guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,7 +4,11 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== postgresql-evr extension no longer required
+
+Installation of the Katello database on remote systems where root access is not available is now possible.
+Only a basic PostgreSQL installation is required, which should enable installs on systems like Amazon RDS or Azure Database for PostgreSQL.
+The contents of postgresql-evr have been moved into Katello itself as a migration, which enables this new capability.
```

Comment:
`postgresql-evr` was a postgresql database extension, which required installation as root. Now, we're shipping the contents of that postgresql database extension within the database migration files that live in Katello's codebase. Due to that, users no longer need to be root to enable the `evr` postgresql type.

If it might be confusing for users, I can keep line 11 out. I added it just for a little extra information.

Reply:
I'll remove it.